tools/add_triage_signature.py

```diff
@@ -878,7 +878,7 @@ class MediaDisconnectionSignature(ErrorSignature):
     Eventually, the signature will include common i/o errors and the number of times they appeared
     '''
 
-    ERRORS_PATTERNS = 'media was likely disconnected'
+    ERRORS_PATTERNS = 'was either disconnected'
```

Comment:
Maybe we should look for ``Unable to read from the discovery media`` instead?

Reply:
sure, makes sense.